haystack/management/commands/clear_index.py

```diff
@@ -10,6 +10,7 @@
 
 class Command(BaseCommand):
     help = "Clears out the search index completely."
+    stealth_options = ('batchsize', 'workers')
```

Comment:
Hey @acdha ...I don't see batchsize and workers used at all here.  Are they cruft from it's history?  Still needed?